specification/synapse/resource-manager/Microsoft.Synapse/preview/2021-06-01-preview/bigDataPool.json

```diff
@@ -606,6 +606,39 @@
           "description": "The filename of the library requirements file."
         }
       }
+    },
+    "SparkConfigProperties": {
+      "type": "object",
+      "description": "SparkConfig Properties for a Big Data pool powered by Apache Spark",
+      "title": "Spark pool Config Properties",
+      "properties": {
+        "time": {
+          "type": "string",
+          "format": "date-time",
+          "readOnly": true,
+          "description": "The last update time of the spark config properties file."
+        },
+        "content": {
+          "type": "string",
+          "description": "The spark config properties."
+        },
+        "filename": {
+          "type": "string",
+          "description": "The filename of the spark config properties file."
+        },
+        "type": {
+          "enum": [
+            "File",
+            "Artifact"
+          ],
+          "type": "string",
+          "description": "The type of the spark config properties file.",
+          "x-ms-enum": {
+            "name": "type",
```

Comment:
1.  If this is really an enum, please list all possible values here.
2. We cannot use "type" as enum name, can we use something like "SparkConfigPropertyFileType" instead?

Reply:
Updated the property name as suggested